apps/test_streaming_scenario.cpp

```diff
@@ -285,18 +317,16 @@ void build_incremental_index(const std::string &data_path, const uint32_t L, con
         delete_tasks[delete_tasks.size() - 1].wait();
 
     std::cout << "Time Elapsed " << timer.elapsed() / 1000 << "ms\n";
-    const auto save_path_inc =
-        get_save_filename(save_path + ".after-streaming-", active_window, consolidate_interval, max_points_to_insert);
+
     index->save(save_path_inc.c_str(), true);
 
     diskann::aligned_free(data);
 }
 
 int main(int argc, char **argv)
 {
-    std::string data_type, dist_fn, data_path, index_path_prefix;
-    uint32_t insert_threads, consolidate_threads;
-    uint32_t R, L, num_start_pts;
+    std::string data_type, dist_fn, data_path, index_path_prefix, label_file, universal_label, label_type;
```

Comment:
Is an index built with filters backward compatible for searching without filters?

Reply:
Yes, it is and I have added dynamic-labels.yml work flow to test just that.